Monika After Story/game/script-moods.rpy

```diff
@@ -939,3 +939,33 @@ label mas_mood_grateful_gratefulforyou:
     m 3ekbla "I love you, [player]~"
     $ mas_ILY()
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_panicking",
+            prompt="...like panicking.",
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_panicking:
+    m 1wkd "Oh no [player]!"
+    m 1rkc "Hm... What could we do..."
+    m 1wub "Oh! What about a breathing excercise?"
+    m 7wua "Let's do it together, breathing in and out."
+    m 2dud "Breathe in.{w=1.0}.{w=1.0}.{w=1.0}."
+    m "And out.{w=0.1}.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}."
+    m 2fua "Let's do it one more time."
+    m 2dud "Breathe in.{w=1.0}.{w=1.0}.{w=1.0}."
+    m "And out.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}."
```

Comment:
```
    m 2dud "Breathe in.{w=1.0}.{w=1.0}.{w=1.0}."
    m "And out.{w=0.1}.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}."
    m 2fua "Let's do it one more time."
    m 2dud "Breathe in.{w=1.0}.{w=1.0}.{w=1.0}."
    m "And out.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}.{w=1.0}."
```

Personally, I think these lines should end with `{nw}`, so players can focus on taking a deep breath without clicking.